timed/subscription/notify_admin.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+
+from django.conf import settings
+from django.core.mail import EmailMultiAlternatives, get_connection
+from django.template.loader import get_template, render_to_string
+
+from timed.projects.models import Project
+
+
+def prepare_and_send_email(request):
```

Comment:
This is business logic, not request handling. Avoid passing in the request object. Instead, pass in the project model directly. This way, the function can be run from other contexts as well if needed

Reply:
Had to pass order duration as well